lib/solaar/ui/window.py

```diff
@@ -625,22 +608,22 @@ def _update_device_panel(device, panel, buttons, full=False):
 		panel._battery._icon.set_sensitive(False)
 		panel._battery._icon.set_from_icon_name(icon_name, _INFO_ICON_SIZE)
 		panel._battery._text.set_sensitive(True)
-		panel._battery._text.set_markup('<small>%s</small>' % _("unknown"))
+		panel._battery._text.set_markup(_("<small>unknown</small>"))
 	else:
 		charging = device.status.get(_K.BATTERY_CHARGING)
 		icon_name = _icons.battery(battery_level, charging)
 		panel._battery._icon.set_from_icon_name(icon_name, _INFO_ICON_SIZE)
 		panel._battery._icon.set_sensitive(True)
 
 		if isinstance(battery_level, _NamedInt):
-			text = _(str(battery_level))
+			text = _("%(battery_level)s") % { 'battery_level': _(str(battery_level)) }
```

Comment:
I see, then wouldn't `_(str(battery_level))` be sufficient? (not sure if that gives sufficient context, not very familiar with this)